FAQ update for Payment Request (#1088)
btcpayserver/btcpayserver-doc: docs/FAQ/Stores.md

```

The issue can be fixed by [selecting a different rate source provider](./Stores.md#how-to-change-the-exchange-rate-provider-for-invoices) in Stores > Settings > Rates, or by [updating your BTCPay Server](./ServerSettings.md#how-to-update-btcpay-server) if you're running version 1.0.3.146 or older. The update will automatically replace Coinaverage with CoinGecko.

## What is a Payment Request? 

> You might have done specific work and want to invoice the customer in BTC. 
> Before the Payment request option, it would've been something like this; 

Reviewer: ```suggestion
A payment request allows you to invoice your clients and get paid in bitcoin, just by sending a simple payment link. If you were to invoice a client by simply using any bitcoin wallet, the process usually looks like this:
```

    Customer: Can I pay the 100$ in bitcoin? 

>     You: Sure you can! Here is my BTC address, and the amount in BTC is 0.0025481 (100$ at the time I wrote the msg). 

Reviewer: Since this is a formal doc, please use `message` instead of `msg` just to make it a bit cleaner in terms of abbreviations we use.
```suggestion
    You: Sure you can! Here is my BTC address, and the amount in BTC is 0.0025481 (100$ at the time of sending the message). 
```

>     Costumer; Ohh, I totally missed your msg, my wallet says 0.0028 now, and I got to get extra as I put this in yesterday. Can I pay tomorrow? 

Reviewer: ```suggestion
    Customer; Ohh, I totally missed your message, my wallet says 0.0028 now, and I need to buy more bitcoin. Can I pay you tomorrow? 
```
```suggestion
    Costumer; Ohh, I totally missed your msg, my wallet says 0.0028 now, and I got to get extra as I put this in yesterday. Can I pay tomorrow? 
```

>     You: Oke, but we have to go through this again tomorrow.

Reviewer: ```suggestion
    You: Okay, but we have to go through this process again tomorrow.
```

> Instead, what you could do with BTCPay server, is use the [Payment Request](../PaymentRequests.md) function.

Reviewer: ```suggestion
BTCPay Server built-in  [Payment Request](../PaymentRequests.md) functionality allows you to get paid by sharing a simple link with your customers/clients.
```

> You only have to create the Payment request and send the counterparty the link generated with the payment request. 
> He will see the requested amount is $ and can immediately click Pay invoice.

Reviewer: ```suggestion
The customer gets a nicely designed payment request page which allows them to pay you at their convenience, where as the bitcoin conversion rate will always be up to date with the currency of your choice
```

> When the invoice has been paid, the invoice is easy to print for bookkeeping purposes. 

Reviewer: ```suggestion
When the payment is made customers can print out the payment request for bookkeeping purposes.
```

[![BTCPay Server Payment Requests](https://img.youtube.com/vi/j6CvwDPvfzQ/mqdefault.jpg)](https://www.youtube.com/watch?v=j6CvwDPvfzQ "BTCPay Server Payment Requests")

> For more examples or detailed information, continue to read on [Payment Request](../PaymentRequests.md)

Reviewer: ```suggestion
You can learn more about this functionality on a dedicated [payment request page](../PaymentRequests.md)
```

## What is the Difference between Payment Request and Invoice? 

An invoice is a document issued by the seller to a buyer to collect payment.

> In BTCPay Server, an invoice represents a document that must be paid within a defined time interval at a fixed exchange rate. Invoices have expiration because they lock the exchange rate within a specified time frame to protect the receiver from price fluctuations.

Reviewer: ```suggestion
In BTCPay Server, an invoice represents a document that must be paid within a defined time interval at a fixed exchange rate. **Invoices have expiration** because they lock the exchange rate within a specified time frame to protect the receiver from price fluctuations.
```

> The opposite is true for a Payment Request. 

Reviewer: ```suggestion
```

> You can Specify a time span for the payment to be payed, the `Payment request` will take into consideration the price fluctuations and does not lock in a exchange rate.
> It also allows you to set the invoice to be paid in parts. 

Reviewer: ```suggestion
Unlike invoices which have a fixed conversion rate, you can think of payment requests as an invoice that doesn't expire and when generated by a payer, always provides the current rate.
```
